internal/cli/projects/apikeys/create.go

```diff
@@ -77,7 +77,10 @@ func CreateBuilder() *cobra.Command {
 		},
 		Args: require.NoArgs,
 		Example: `  # Create an organization API key with the GROUP_OWNER role and assign it to the project with ID 5e2211c17a3e5a48f5497de3:
-  atlas projects apiKeys create --desc "My API key" --projectId 5e1234c17a3e5a48f5497de3 --role GROUP_OWNER --output json`,
+  atlas projects apiKeys create --desc "My API key" --projectId 5e1234c17a3e5a48f5497de3 --role GROUP_OWNER --output json
+  
+  # Create an organization API key with the GROUP_SEARCH_INDEX_EDITOR and GROUP_DATABASE_ACCESS_ADMIN roles and assign it to the project with ID 5e2211c17a3e5a48f5497de3:
```

Comment:
```suggestion
  # Create a project API key with the GROUP_SEARCH_INDEX_EDITOR and GROUP_DATABASE_ACCESS_ADMIN roles and assign it to the project with ID 5e2211c17a3e5a48f5497de3:
```

Reply:
This command does create an org level API key and just assigns it to a project. Follows the endpoint: [Create and Assign One Organization API Key to One Project](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Programmatic-API-Keys/operation/listProjectApiKeys)

Reply:
yes but no, all project API keys are an org API key and not all org API keys are project API keys.

When creating a project API key (this command) it's also assigned to the org as an "org member" but org member has the lowest org permissions, so if you want to be complete about it  
```suggestion
  # Create a project API key with the GROUP_SEARCH_INDEX_EDITOR and GROUP_DATABASE_ACCESS_ADMIN roles in the project with ID 5e2211c17a3e5a48f5497de3, the API key will also be added to the project parent org as an organization member:
```

Reply:
This command also allows you to specify the org role as well but the default behaviour is "org member" as you say. 

Atlas-sdk and api docs also refer to this as creating an org api key assigned to a project.
I think it would be better to keep the example description but I will amend the example to include specifying org roles. I'll also add a note to the command description that the default behaviour is to assign the parent org as org member:
``` suggestion
  # Create an organization API key with the ORG_OWNER and GROUP_SEARCH_INDEX_EDITOR roles and assign it to the project with ID 5e2211c17a3e5a48f5497de3:
atlas projects apiKeys create --desc "My API key" --projectId 5e1234c17a3e5a48f5497de3 --role ORG_OWNER,GROUP_SEARCH_INDEX_EDITOR --output json`,
```
Also
```
Long: `<...>
By default, the specified project's parent organization will be assigned as organization memeber.
```
@gssbzn  WDYT?